docs/language/overview.md

```diff
@@ -1639,7 +1615,7 @@ Here the lateral scope, described below, creates a subquery
 ```
 yield {name,elem:this}
 ```
-for each sub-sequence of values derived from each outer input value.
+for each subsequence of values derived from each outer input value.
```

Comment:
When I started the proofreading, the count on "subsequence" was 3 and "sub-sequence" was 2. My dictionary searches also seemed to imply "subsequence" was the more common form.